mmv1/products/looker/Instance.yaml

```diff
@@ -0,0 +1,362 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+base_url: projects/{{project}}/locations/{{region}}/instances
+create_url: projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  A Google Cloud Looker instance.
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 90
+  update_minutes: 90
+  delete_minutes: 60
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'looker_instance_basic'
+    primary_resource_id: 'looker-instance'
+    vars:
+      instance_name: 'my-instance'
+      client_id: 'my-client-id'
+      client_secret: 'my-client-secret'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'looker_instance_full'
+    primary_resource_id: 'looker-instance'
+    vars:
+      instance_name: 'my-instance'
+      client_id: 'my-client-id'
+      client_secret: 'my-client-secret'
+parameters:
+  # TODO: resourceref?
+  - !ruby/object:Api::Type::String
+    name: 'region'
+    description: |
+      The name of the Looker region of the instance.
+    required: false
+    immutable: true
+    url_param_only: true
+    ignore_read: true
+    default_from_api: true
+properties:
+  # Admin Settings Object
+  - !ruby/object:Api::Type::NestedObject
+    name: adminSettings
+    description: |
+      Looker instance Admin settings.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'allowedEmailDomains'
+        item_type: Api::Type::String
+        description: |
+          Email domain allowlist for the instance.
+
+          Define the email domains to which your users can deliver Looker (Google Cloud core) content.
+          Updating this list will restart the instance. Updating the allowed email domains from terraform
+          means the value provided will be considered as the entire list and not an amendment to the
+          existing list of allowed email domains.
+  # Admin Settings Object - End
+  - !ruby/object:Api::Type::String
+    name: consumerNetwork
+    description: |
+      Network name in the consumer project in the format of: projects/{project}/global/networks/{network}
+      Note that the consumer network may be in a different GCP project than the consumer
+      project that is hosting the Looker Instance.
+  - !ruby/object:Api::Type::Time
+    name: createTime
+    description: |
+      The time the instance was created in RFC3339 UTC "Zulu" format,
+      accurate to nanoseconds.
+    output: true
+  # Deny Maintenance Period Object
+  - !ruby/object:Api::Type::NestedObject
+    name: denyMaintenancePeriod
```

Comment:
Are `lastDenyMaintenancePeriod`/`maintenanceSchedule` intentionally omitted?

Reply:
Yes, that was intentional. Both are computed properties, that keep changing (maybe good candidate for data resource in the future).